data/scripts/systems/custom_monster_loot.lua

```diff
@@ -0,0 +1,38 @@
+-- Drops custom loot for all monsters
```

Comment:
maybe it would be more interesting to use onSpawn like we do with snake monsters to decrease their life, so we could configure loots for events or things like that.